srcpkgs/bazel-buildtools/template

```diff
@@ -0,0 +1,35 @@
+# Template file for 'bazel-buildtools'
+pkgname=bazel-buildtools
+version=5.1.0
+revision=1
+wrksrc="buildtools-${version}"
+hostmakedepends="go"
```

Comment:
```suggestion
build_style=go
go_import_path="github.com/bazelbuild/buildtools"
go_package="${go_import_path}/unused_deps ${go_import_path}/buildifier
 ${go_import_path}/buildozer"
```

Reply:
Oh I was not able to make it works properly at first because I did not use go_package, let me try!

Reply:
This works fine! I tested the change on the same platforms as stated in the original message

thanks